src/rtk/features/delegationRequest/delegationRequestSlice.ts

```diff
@@ -0,0 +1,86 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+import type { DelegableApi } from '../delegableApi/delegableApiSlice';
+import type { DelegableOrg } from '../delegableOrg/delegableOrgSlice';
+
+export interface ApiDelegation {
+  orgName: string;
+  apiName: string;
+}
+
+export interface InitialState {
+  loading: boolean;
+  error: string;
+  succesfulApiDelegations: ApiDelegation[];
+  failedApiDelegations: ApiDelegation[];
+}
+
+export interface DelegationRequest {
+  apiIdentifier: string;
+  orgNr: string;
+}
+
+const initialState: InitialState = {
+  loading: true,
+  error: '',
+  succesfulApiDelegations: [
+    {
+      orgName: 'Org blabla',
+      apiName: 'Api blabla',
+    },
+  ],
+  failedApiDelegations: [
+    {
+      orgName: 'Org blabla',
+      apiName: 'Api blabla',
+    },
+  ],
+};
+
+export const delegateApi = async (
```

Comment:
I'm assuming this is made for later use? Cause I can't see that it's used anywhere as of now

Reply:
Is going ot be used for later. Needed to write that code to solve a potential hard problem.